### PR TITLE
Drive AUD traffic to support platform

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?
Drive all the traffic to support platform and stop sending people to contributions. Additionally, the two button template is removed.
## Screenshots

|       Country Group Name        | Epic | Engagement Banner |
|---------------|------|-------------------|
| **GBPCountries**  |      picture 701     |   picture 700   |
| **USDCountries**            |  picture 702   |         picture 703         |
| **Canada**        |  picture 704   |   picture 705      |
| **NZDCountries**     |  picture 708    | picture 709 |
| **AUDCountries**     |  picture 699    |     picture 698             |
| **International** |  picture 706    |   picture 707   |
| **EURCountries** |  picture 711       |picture 710   |  

## Tested in CODE?
Not yet

### CODE REVIEW

**Comment by svillafe** [comment]
Tested in CODE (all the screenshots are from there)

**Review by ionamckendrick — APPROVED** [review]
Looks good!

File: static/src/javascripts/projects/common/modules/commercial/support-utilities.js [hunk 1]
```diff
@@ -1,22 +1,5 @@
 // @flow
-import {
-    getSync as geolocationGetSync,
-    getSupporterCountryGroup as geolocationGetSupporterPaymentRegion,
-} from 'lib/geolocation';
+const supportContributeURL = 'https://support.theguardian.com/contribute';
+const supportSubscribeURL = 'https://support.theguardian.com/contribute';
```
Comment (Mullefa) [inline static/src/javascripts/projects/common/modules/commercial/support-utilities.js]: should this be `/subscribe`?

**Review by Mullefa — COMMENTED** [review]
One question / comment. Asides from that - looks good!

File: static/src/javascripts/projects/common/modules/commercial/support-utilities.js (hunk 1 above)
Comment (svillafe) [reply static/src/javascripts/projects/common/modules/commercial/support-utilities.js]: Yes! good catch @Mullefa 🥇

**Review by Mullefa — APPROVED** [review]

**Comment by svillafe** [comment]
Tested in code and is working.

**Pull request merged**